addon/templates/components/uni-input.hbs

```diff
@@ -0,0 +1,17 @@
+{{input
+  type=type
+  value=computedValue
+  placeholder=placeholder
+  disabled=disabled
+  name=name
+  maxlength=maxLength
+  class=(i "${default} ${custom} ${success} ${error} ${warning}"
```

Comment:
This could be avoided by using classNameBindings if it's created a more "pure" uni-input without the tooltip but I don't know if we really want that...